manifests/state-nv-ipam-cni/030-controller-config.yaml

```diff
@@ -0,0 +1,21 @@
+# 2023 NVIDIA CORPORATION & AFFILIATES
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nv-ipam-controller-config
+  namespace: {{ .RuntimeSpec.Namespace }}
+data:
+  controller-conf: |
```

Comment:
in Yury code it is config https://github.com/Mellanox/nvidia-k8s-ipam/pull/5/files#diff-5c4df879eb29e6a4f6acc02753175b229a48f72c6b725a75cc00e9ab46bc33a2R29